server/gitrest/packages/gitrest-base/src/utils/redisFs/redis.ts

```diff
@@ -52,7 +52,7 @@ export class Redis {
 	}
 
 	public async delete(key: string): Promise<boolean> {
-		const result = await this.client.del(this.getKey(key));
+		const result = await this.client.del(key);
```

Comment:
Just want to make sure that this change is intentional. Previously we were using thig.getKey(key).

Reply:
This is intentional. In the redis Fs, the pattern that we are using is that we have a particular folder path for a doc and we want to delete all the values related to that doc. While we enumerate what we want to delete (like using redis keys*), we get the keys with the prefix. If we getKey again here, it would prefix the key twice. Also, this method is not being used anywhere else, so it is a safe change.

Reply:
I agree that this is safe, since this Redis class is within `redisFs/`, but for the sake of future maintenance, could you add a quick code comment explaining like you did in this PR comment?

Reply:
Just to clarify: earlier, we would delete the key in Redis after adding the appropriate prefix.

now that we just call `this.client.del(key)`, will the command succeed? Are we not setting the key with the prefix every time?

I might be failing to understand the intention here. Or maybe, is it that we always call `delete()` after enumerating the keys, which naturally come with a prefix and therefore we don't need to add the prefix again?

Reply:
Modified this to make the caller pass in the param if we want to ignore the prefixing. Hope this makes it more clear.